Provide sample command to upload training data (#16)
microsoft/onnxruntime-training-examples: nvidia-bert/README.md

    * Register the blob container as a data store
    * Mount the data store in the compute targets used for training

    Please refer to the [storage guidance](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-access-data#storage-guidance) for details on using Azure storage account for training in Azure Machine Learning. 

>     With the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest), the data folder can be transferred to an Azure blob storage using

Reviewer: Could we move this up into the bullet where we talk about uploading?

>     With the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest), the data folder can be transferred to an Azure blob storage using

Reviewer: ```suggestion
    Transfer the data using to Azure using the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) with the command:
```
